### PR TITLE
[WIP] prevent built node and run loadflow when action in progress in the hypotheses

### PR DESCRIPTION
**Pull request by bendaamerahm**
prevent built current node and run loadflow when action in progress in the hypotheses

### CODE REVIEW

File: src/components/run-button.js [hunk 1]
```diff
@@ -132,11 +132,14 @@ const RunButton = (props) => {
 
     let buttonDisabled =
         props.disabled ||
+        props.hypothesisInLoad ||
```
Comment (sBouzols) [inline src/components/run-button.js:135]: remove this line

File: src/components/run-button.js [hunk 2]
```diff
@@ -132,11 +132,14 @@ const RunButton = (props) => {
...
         (selectedIndex === 0 && getRunningStatus() !== RunningStatus.IDLE) ||
         (selectedIndex === 1 && isRunning());
 
     let selectionDisabled =
-        props.disabled || (selectedIndex === 0 && isRunning());
+        props.disabled ||
+        props.hypothesisInLoad ||
```
Comment (sBouzols) [inline src/components/run-button.js:141]: remove

**Review by sBouzols — COMMENTED** [review]
Try to use notificationIdList instead of 2 new notifications consumer

**Comment by bendaamerahm** [comment]
old PR => create new branche then new PR

**Pull request closed without merging**